test_cursor:test_copy_with_file testcase fix:

tudit opened the pull request:
-test case fails on windows platform (probably fails on few other platforms)
-On a few platforms, tempfile.TempFile() creates file-like wrapper objects that contain the file objects causing the cursor to raise an exception
-Added the same check that returns the temp file object to get the file object (for the check refer tempfile module: line 540)
-file objects haven't been properly closed, causing the temp files to persist. Fixed this by moving the file handling logic to context manager

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/uber/vertica-python?pullRequest=164) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/uber/vertica-python?pullRequest=164) before we can accept your contribution.<br/><hr/>**tumuluru** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/uber/vertica-python?pullRequest=164) it.</sub>

tudit commented:
test_cursor:test_copy_with_file testcase fix:
-test case fails on windows platform (probably fails on few other platforms)
-On a few platforms, tempfile.TempFile() creates file-like wrapper objects that contain the file objects causing the cursor to raise an exception
-Added the same check that returns the temp file object to get the file object (for the check refer tempfile module: line 540)
-file objects haven't been properly closed, causing the temp files to persist. Fixed this by moving the file handling logic to context manager
